src/base/tz_support.c

```diff
@@ -5469,3 +5469,16 @@ conv_tz (void *p_out, const void *p_in, DB_TYPE type)
 
   return err_status;
 }
+
+/*
+ * tz_get_offset_in_mins () - time zone offset in minutes from GMT
+ - gmtime(0) returns the GMT time of the second ZERO of the epoch
+ - mktime() simply translate the above moment into equivalent local time
+ - the diff between these 2 values gives the timezone offset in seconds (but keep in mind that one of them is ZERO)
+ */
+int
+tz_get_offset_in_mins ()
+{
+  time_t tt0 = 0;
+  return (int) (mktime (gmtime (&tt0)) / -60);
```

Comment:
This is used in db_sys_timezone to return current timezone (as in moment of call).
The  code of tz_get_offset_in_mins returns the timezone as it was in 1970.
We may find examples (locale, and current time) were the tow values are not the same.

Reply:
https://arstechnica.com/civis/viewtopic.php?t=136625

Reply:
As a fun fact there are in fact multiple countries who have changed their timezone since 1970 , Venezuela comes to mind. So I changed the implementation.